trashtalk/run.py

```diff
@@ -28,7 +28,7 @@
         app.logger.info("Production server configured ...")
     else:
         app = app_factory('trashtalk.settings.Development')
-        # app.config.from_pyfile('dev.cfg')
+        app.config.from_pyfile('dev.cfg')
```

Comment:
:+1: 
I left it out before to make sure I didn't break the app for you :) Thanks for updating!

Reply:
I actually need your help getting it to work. Everything that relies on dev.cfg is broken.